src/fonts-reset-var/normalize.scss

```diff
@@ -0,0 +1,176 @@
+.header,
+.menu {
+  font-family: 'Space Grotesk', serif;
+}
+
+h2{
+font-family: 'Space Grotesk', serif;
+font-weight: 700;
+font-size: 32px;
+line-height: 32px;
+}
+
+.header__title{
```

Comment:
Starting from this line, styles in `normalize.scss` correspond to different BEM blocks and are not related to normalization - so they have to be moved to separate file(s).